release-notes/7.0/known-issues.md

```diff
@@ -98,6 +98,15 @@ More information and workaround can be found at https://github.com/dotnet/runtim
 
 ## .NET SDK
 
+### [7.0] Projects using certain workloads don't load, build, and or run if .NET 7 Preview SDK workloads are installed: 
+
+If a preview 7 .NET SDK is installed, projects with workload dependencies such as `microsoft.net.workload.mono.toolchain` may fail to build, load, or run. An example of this issue is described in https://github.com/dotnet/sdk/issues/28947 and also here: https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html. 
+
+**Workarounds**
+The best method to resolve the issue is to uninstall any .NET 7 preview SDKs. On Windows, this can be done easily with add/remove programs. Another option is to try deleting the folder `C:\Program Files\dotnet\sdk-manifests\microsoft.net.workload.mono.toolchain`. Dotnet-core-uninstall https://github.com/dotnet/cli-lab/releases is another option for uninstalling the .NET 7 preview SDKs.
```

Comment:
```suggestion
The best method to resolve the issue is to uninstall any .NET 7 preview SDKs. On Windows, this can be done easily with add/remove programs. Another option is to try deleting the folder `C:\Program Files\dotnet\sdk-manifests\microsoft.net.workload.mono.toolchain`. [Dotnet-core-uninstall](https://github.com/dotnet/cli-lab/releases) is another option for uninstalling the .NET 7 preview SDKs.
```